config/sql/migration/migrate.15.sql

```diff
@@ -0,0 +1,7 @@
+-- Don't modify this! Create a new migration instead--see docs/schema-migrations.adoc
+BEGIN TRANSACTION;
+
+CREATE TABLE migrations(migration TEXT NOT NULL, version INT NOT NULL);
+DROP TABLE version;
```

Comment:
Do we really need to stop having the version directly in the db and compute it from the content of `migrations`? It's complicating the logic significantly.